sql/pg-language.sql

```diff
@@ -161,9 +147,7 @@ CREATE OPERATOR <> (
 COMMENT ON OPERATOR <>(language, language) IS 'not equal';
 
 CREATE FUNCTION language_cmp(language, language)
-    RETURNS int4
-    AS '$libdir/language.so'
-    LANGUAGE C IMMUTABLE STRICT;
+RETURNS integer LANGUAGE internal IMMUTABLE AS 'btcharcmp';
```

Comment:
I really like this convention.  I was thinking about suggesting it for the other extensions but wanted to get other things done first.

Reply:
In fact we could take it further and just use existing internal functions for operators if they are already registered?

Reply:
(We don't need to have our own functions at all)